app/views/submissions/_judging_status.html.erb

```diff
@@ -0,0 +1,60 @@
+<div data-controller="submission-details">
+    <%= form_with(model: @submission, url: submission_path(@submission), class: "maxw-mobile-lg") do |form| %>
+        <div class="usa-checkbox" id="eligible-for-evaluation">
+            <input
+            class="usa-checkbox__input usa-checkbox__input--tile"
+            id="judging-status-selected"
+            type="checkbox"
+            name="submission[judging_status]"
+            value="selected"
+            <%= if @submission.judging_status == "winner" then "disabled" end %>
+            <%= if @submission.judging_status.in?(["selected", "winner"]) then "checked" end %>
+            data-action="change->submission-details#selectedSubmit"
+            />
+            <label class="usa-checkbox__label" for="judging-status-selected"
+            >This submission is eligible for evaluation.
+            <span class="usa-checkbox__label-description"
+                >Check the box if the submission is pre-screened and is eligible for the evaluation process.</span
+            ></label
+            >
+        </div>
+    <% end %>
+    <%= form_with(model: @submission, url: submission_path(@submission), class: "maxw-mobile-lg") do |form| %>
+        <div class="usa-checkbox" id="selected-to-advance">
+            <input
+            class="usa-checkbox__input usa-checkbox__input--tile"
+            id="judging-status-winner"
+            type="checkbox"
+            name="submission[judging_status]"
+            value="winner"
+            <%= unless @submission.judging_status.in?(["selected", "winner"]) then "disabled" end %>
+            <%= if @submission.judging_status == "winner" then "checked" end %>
```

Comment:
how do you feel about the postfix version of conditionals like this?
```suggestion
            <%= "disabled" unless @submission.judging_status.in?(["selected", "winner"]) %>
            <%= "checked" if @submission.judging_status == "winner" %>
```

Reply:
love it